dump_manager_bmc.cpp

```diff
@@ -166,7 +169,29 @@ void Manager::checkAndCreateCoreDump()
             log<level::INFO>(
                 fmt::format("Core file found, files size {}", files.size())
                     .c_str());
-            captureDump(Type::ApplicationCored, files);
+
+            try
+            {
+                captureDump(Type::ApplicationCored, files);
+            }
+            catch (const QuotaExceeded& e)
+            {
+                log<level::ERR>(
+                    fmt::format(
+                        "checkAndCreateCoreDump:: Exception Caught while capturing dump: "
+                        "{} REASON: {}",
+                        e.name(), e.description())
+                        .c_str());
```

Comment:
We should return here since no further dump creation is possible

Reply:
Sure.

Reply:
But there is nothing after that. So anyways it will return. Do you still want a return statement here?

Reply:
Or would it better to handle the exception in the main? https://github.com/ibm-openbmc/phosphor-debug-collector/blob/1020/dump_manager_main.cpp#L98

Reply:
It is better to handle here, main doesn't need to know about the difference between other exceptions and quota exceeded,